### PR TITLE
 Removed deprecated calls

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is a patch.

## Subject

Removed some more deprecated calls.

### CODE REVIEW

**Comment by OskarStark** [comment]
Thank you @core23 👍

File: Form/Type/TemplateChoiceType.php [hunk 1]
```diff
@@ -73,7 +73,10 @@ public function getTemplates()
      */
     public function getParent()
     {
-        return 'choice';
+        // NEXT_MAJOR: Remove ternary (when requirement of Symfony is >= 2.8)
+        return method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+            ? 'Symfony\Component\Form\Extension\Core\Type'
```
Comment (jlamur) [inline Form/Type/TemplateChoiceType.php]: @core23 Fix it please :)

File: Form/Type/PageTypeChoiceType.php [hunk 2]
```diff
@@ -76,7 +76,10 @@ public function getPageTypes()
      */
     public function getParent()
     {
-        return 'choice';
+        // NEXT_MAJOR: Remove ternary (when requirement of Symfony is >= 2.8)
+        return method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+            ? 'Symfony\Component\Form\Extension\Core\Type'
```
Comment (jlamur) [inline Form/Type/PageTypeChoiceType.php]: @core23 This one too.

File: Form/Type/TemplateChoiceType.php (hunk 1 above)
Comment (jlamur) [reply Form/Type/TemplateChoiceType.php]: cf. https://github.com/sonata-project/SonataPageBundle/pull/870#pullrequestreview-67109486

**Pull request merged**